### PR TITLE
first version

### PR DESCRIPTION
**Pull request by Harsh-D**
Kindly review and give your feedback

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// var ip = document.querySelector("#ip");
```
Comment (bulsaraharshil) [inline app.js:1]: Remove unnecessary code, though it is commented still this is not a good practice to have

File: app.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+// var op = document.querySelector("#op");
```
Comment (bulsaraharshil) [inline app.js]: Remove unnecessary code, though it is commented but it is not a good practice to follow

Comment (Harsh-D) [reply app.js]: Issue resolved.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+.nav {
+    grid-area: nav;
+    border: 1px ;
+    padding: 1em;
+    margin: 1em;
+}
+
+.main {
+    grid-area: main;
+    /* border: 1px solid; */
```
Comment (OmkarArora) [inline style.css]: Remove comment if not being used

File: style.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+.footer {
+    grid-area: footer;
+    border: 1px ;
+    padding: 1em;
+    margin: 1em;
+}
+
+.footer {
+    /* margin-top: 1rem; */
```
Comment (OmkarArora) [inline style.css]: Remove comment if not being used

File: style.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+        "nav main main main main"
+        "nav main main main main"
+        "footer footer footer footer footer";
+    display: grid;
+    gap: rem;
+}
+
+.nav {
+    grid-area: nav;
+    border: 1px ;
```
Comment (OmkarArora) [inline style.css:32]: Remove these random spaces before semicolon, present in few more lines

File: app.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+
+
+//Toast
+let myToastButton = document.getElementById("myToastButton");
+let myHideToastButton = document.getElementById("myHideToastButton");
+let myToastClass = document.getElementsByClassName("toast")[0];
+
+myToastButton.onclick = function(){
+    console.log("clicked");
```
Comment (OmkarArora) [inline app.js]: Remove console log

File: style.css [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+.layout {
+    grid-template-areas:
+        "header header header header header"
+        "nav main main main main"
+        "nav main main main main"
+        "nav main main main main"
+        "footer footer footer footer footer";
+    display: grid;
+    gap: rem;
```
Comment (OmkarArora) [inline style.css]: No value given here, only unit

**Comment by OmkarArora** [comment]
In the website sidebar navigation, only the text is clickable, make the whole pill/button clickable. In my opinion it will give better UX. 
Rest, the design looks good.

File: style.css (hunk 5 above)
Comment (Harsh-D) [reply style.css:32]: done

File: app.js (hunk 6 above)
Comment (Harsh-D) [reply app.js]: done

File: style.css (hunk 3 above)
Comment (Harsh-D) [reply style.css]: done

File: style.css (hunk 4 above)
Comment (Harsh-D) [reply style.css]: done

**Comment by Harsh-D** [comment]
> In the website sidebar navigation, only the text is clickable, make the whole pill/button clickable. In my opinion it will give better UX.
> Rest, the design looks good.

Okay, will try to improve that

File: app.js [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+// Get the button that opens the modal
+let btn = document.getElementById("myModalBtn");
+
+// Get the <span> element that closes the modal
+let span = document.getElementsByClassName("close")[0];
+
+// When the user clicks on the button, open the modal
+btn.onclick = function () {
+    console.log("modal");
```
Comment (theniteshnarang) [inline app.js]: You can remove this console.

File: lognui.css [hunk 9]
```diff
@@ -0,0 +1,599 @@
...
+
+.input-error-div {
+    display: flex;
+    flex-direction: column;
+    gap: 0.2rem;
+}
+
+.input {
+    width: 50%;
+    max-width: 70%;
```
Comment (theniteshnarang) [inline lognui.css]:
```suggestion
    max-width: 50%;
```
max-width is overriding above width property, remove that width property and do the changes as suggested.

File: lognui.css [hunk 10]
```diff
@@ -0,0 +1,599 @@
...
+    color: rgba(184, 14, 14, 0.692);
+    font-weight: bold;
+    padding: 0.1rem 0 0 0.5rem;
+}
+
+/* Input section : END*/
+/* List section : START*/
+.list-div {
+    display: flex;
+    flex-direction: row;
```
Comment (theniteshnarang) [inline lognui.css]: ```display:flex``` is by default set as ```row```, you don't have to explicitly specify its direction.

File: lognui.css [hunk 11]
```diff
@@ -0,0 +1,599 @@
...
+    overflow: auto;
+    background-color: rgb(0, 0, 0);
+    background-color: rgba(0, 0, 0, 0.4);
+}
+
+.modal-content {
+    background-color: var(--primary-color);
+    color: #151d2e;
+    margin: 15% auto;
+    /* 15% from the top and centered */
```
Comment (theniteshnarang) [inline lognui.css]: There's no need for such explanatory comments.

File: lognui.css [hunk 12]
```diff
@@ -0,0 +1,599 @@
...
+}
+
+.card-badge-div {
+    grid-area: card-badge-div;
+    position: relative;
+}
+
+.card-dismiss-div {
+    grid-area: card-dismiss-div;
+    position: relative;
```
Comment (theniteshnarang) [inline lognui.css]: You don't need to specify ```position:relative``` here, It makes no difference if I remove it. Please correct it everywhere.

File: style.css [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.footer {
+    grid-area: footer;
+    border: 1px ;
+    padding: 1em;
+    margin: 1em;
+}
+
+.footer {
```
Comment (theniteshnarang) [inline style.css]: Why two-footer selectors? See line 53

**Review by theniteshnarang — CHANGES_REQUESTED** [review]
Hey Harsh,

Wonderfully written code and good practices followed, keep doing the good work.

I have found a few nitpicks, which you can look into.

**Comment by Harsh-D** [comment]
> Hey Harsh,
> 
> Wonderfully written code and good practices followed, keep doing the good work.
> 
> I have found a few nitpicks, which you can look into.

Thank you for the review Nitesh, I will look into your suggestions

**Comment by Harsh-D** [comment]
Changes made as per suggestion

**Pull request merged**